convert2rhel/main.py

```diff
@@ -183,6 +192,20 @@ def main_locked():
     return 0
 
 
+def _raise_for_skipped_failures(results):
```

Comment:
Maybe I would consider a different name of this function. Like `_check_for_results_skipped_failures`, `raise_when_skipped_failures` or something else. Without the function body I wouldn't understand what is it doing

Reply:
We can change it later, merging for now

Reply:
Agree, it's not a big problem